sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -15,7 +15,7 @@ Maven dependency for Azure Key Client library. Add it to your project's pom file
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-certificates</artifactId>
-    <version>4.0.0-preview.5</version>
+    <version>4.0.0-beta.6</version>
```

Comment:
This is good to update since it looks like it was missed when they released the beta.6 version of azure-security-keyvault-certificates

Reply:
yup, updating the KV READMEs to update latest release everywhere.